docs/en_US/Tutorial/Nnictl.md

```diff
@@ -24,6 +24,7 @@ nnictl support commands:
 * [nnictl package](#package)
 * [nnictl ss_gen](#ss_gen)
 * [nnictl --version](#version)
+* [nnictl export_results](#export-results)
```

Comment:
what is the difference of this command from `nnictl experiment export` in https://nni.readthedocs.io/en/latest/Tutorial/Nnictl.html#manage-experiment-information

Reply:
It dumps not only experiment settings, trial final results, but also all intermediate results.

Reply:
we should combine the commands, otherwise, the commands are messy. please try to merge your command to `nnictl experiment export`

Reply:
OK, I will add a CLI option to specify whether intermediate results are required or not.

Reply:
remove unused command in doc.